test-util/src/main/java/org/triplea/test/lib/MatchBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.triplea.test.lib;
```

Comment:
`lib` seems redundant.  I would suggest it match whatever suffix you use in the module name (e.g. `util`, `common`, `core`, etc.).